src/NuGet.Core/NuGet.Protocol.Core.Types/SourceCacheContext.cs

```diff
@@ -12,7 +12,10 @@ namespace NuGet.Protocol.Core.Types
     /// </summary>
     public class SourceCacheContext : IDisposable
     {
-        private static readonly string _tempFolderGuid = Guid.NewGuid().ToString();
```

Comment:
/cc @emgarten, this change seems good to me. Do you know why this was `static` before?